src/EnergyPlus/PluginManager.cc

```diff
@@ -724,9 +724,41 @@ namespace PluginManagement {
         globalVariableNames.clear();
         globalVariableValues.clear();
         plugins.clear();
-        pluginManager.reset(); // delete the current plugin manager instance, which was created in simulation manager, this clean up Python
         PluginManagement::fullyReady = false;
         PluginManagement::apiErrorFlag = false;
+        PluginManager * p = PluginManagement::pluginManager.release();
```

Comment:
Release the unique pointer ownership and then call delete on the pointer itself.